src/exceptions/LLVM-exception.xml

```diff
@@ -5,6 +5,9 @@
          <crossRef>http://lists.llvm.org/pipermail/llvm-dev/2017-August/116266.html</crossRef>
       </crossRefs>
       <notes>This exception was created specifically to be used with Apache-2.0</notes>
+	  <titleText>---- LLVM Exceptions to the Apache 2.0 License ----
+			<br/>
```

Comment:
I'd rather wrap with `<p>` and drop the `<br/`> to match the bulk of our existing licenses.